### PR TITLE
Webstack with node 7 image

### PR DESCRIPTION
**Pull request by tjomk**
- Install gcloud
- Add new image with node 7

### CODE REVIEW

File: python-uwsgi/Dockerfile [hunk 1]
```diff
@@ -1,18 +1,31 @@
...
+RUN apt-get update && \
     apt-get install --yes --no-install-recommends \
             python-all-dev \
             python-pip \
             python-wheel \
             uwsgi-plugin-python \
+            lsb-release \
     && rm -rf /var/cache/apt
 
+RUN export CLOUD_SDK_REPO="cloud-sdk-$(lsb_release -c -s)" \
```
Comment (rollcat) [inline python-uwsgi/Dockerfile]: Why would you *need* this here? You're bloating a single-purposed base image, that doesn't need this 99.9% of the time

File: python-uwsgi/Dockerfile [hunk 2]
```diff
@@ -1,18 +1,31 @@
-from unit9/base:latest
-maintainer Kamil Cholewiński <kamil.cholewinski@unit9.com>
+FROM unit9/base:latest
```
Comment (rollcat) [inline python-uwsgi/Dockerfile]: Please don't change capitalisation, stick to the current convention (whatever the convention is)

File: webstack-node7/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,21 @@
+from unit9/python-uwsgi:latest
```
Comment (rollcat) [inline webstack-node7/Dockerfile]: Again, inconsistent capitalisation

File: webstack-node7/readme.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+# Base image with Python, uWSGI, Node, Ruby, & frontend tools
+
+See [the readme for `unit9/webstack`](/webstack).
```
Comment (rollcat) [inline webstack-node7/readme.md:3]: Not based on unit9/webstack?

**Review by rollcat — CHANGES_REQUESTED** [review]

File: webstack-node7/readme.md (hunk 4 above)
Comment (tjomk) [reply webstack-node7/readme.md:3]: Nope, it installs node 4

File: shippable/Dockerfile [hunk 5]
```diff
@@ -0,0 +1,34 @@
+FROM unit9/base:latest
```
Comment (tjomk) [inline shippable/Dockerfile:1]: Need this image for building on Shippable. The base image they provide has old Node, and for pushing builds to repo we need awscli/gcloud.

File: webstack-node7/readme.md (hunk 4 above)
Comment (rollcat) [reply webstack-node7/readme.md:3]: Ah yeah, readme from webstack still mostly applies

File: shippable/Makefile [hunk 6]
```diff
@@ -0,0 +1,17 @@
+TAG = unit9/python-uwsgi
```
Comment (rollcat) [inline shippable/Makefile]: wrong tag!

**Review by rollcat — CHANGES_REQUESTED** [review]

**Review by rollcat — APPROVED** [review]

**Pull request merged**